testing/zerotier-one/zerotier-one.initd

```diff
@@ -1,11 +1,14 @@
 #!/sbin/openrc-run
 
-depend() {
-    provide net
-    use network
-    before firewall netmount
-}
-
+name=zerotier-one
 command="/usr/sbin/zerotier-one"
+command_args="$zerotier_opts"
```

Comment:
I'm not sure it's a good idea to have `conf.d` set OpenRC options automatically. It may get wonky if we want to add our own arguments.